lib/event.rb

```diff
@@ -0,0 +1,45 @@
+class Event
+  attr_reader :name, :ages
+
+  def initialize(name, ages)
+    @name = name
+    @ages = ages
+  end
+
+  def max_age
+    @ages.max
+  end
+
+  def min_age
+    @ages.min
+  end
+
+  def average_age
+    self.average(ages)
```

Comment:
When you call methods from within other methods, `self` is implied.  You could refactor this to be:

```
def average_age
  average(@ages)
end
```